index.html

```diff
@@ -380,6 +381,60 @@ <h2>About me</h2>
       </ul>
       <hr class="hrc" />
     </section>
+    <div class="popup-modal" id="overlay">
+      <!-- <div class="popup-content">
+        <div class="modal-card">
+          <div class="popupcancel" data-close-button>
+            <img
+              src="./images/projects/footericons/popupcancel.svg"
+              alt="cancelbutton"
+            />
+          </div>
+          <h2>Multi-Post Stories</h2>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+          </ul>
+          <div class="popupimage">
+            <img src="./" alt="" />
+          </div>
+          <div class="popupara">
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it 1960s. Lorem Ipsum is simply dummy text of
+              the printing and typesetting industry. Lorem Ipsum has been the
+              industry's standard dummy text ever since the 1500s, when an
+              unknown printer took a galley of type and scrambled it 1960s with
+              the releax map lapora verita.
+            </p>
+          </div>
+
+          <div class="buttonstyling">
+            <button>
+              <div class="buttonstyle fixingbutton">
+                <p>See live</p>
+                <img
+                  src="./images/projects/footericons/iconseeLive.svg"
+                  alt="button image"
+                />
+              </div>
+            </button>
+            <button data-modal-target>
+              <div class="buttonstyle">
+                <p>See source</p>
+                <img
+                  src="./images/projects/footericons/button2image.svg"
+                  alt="button image"
+                />
+              </div>
+            </button>
+          </div>
+        </div>
+      </div> -->
```

Comment:
- [ ] Kindly as stated by the previous reviewer, Kindly 😊 try to remove the unnecessary comment here because this makes your codebase ugly ,so we want your code to look nice now and in the next projects try to not put unnecessary comments at the base code to make the code organized and nice for the others😃